Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,31 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            //Start program
+            Console.WriteLine("Hello World! Welcome to the first C# 2020 Bootcamp Homework.");
+            PromptUser();
+            Console.WriteLine("Okay, Person #2:");
+            PromptUser();
+        }
+        static double GetBMI(double weight, int height) //calculates the BMI by receiving weight and height as parameters and dividing weight by height squared (kg/m^2)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename method 'GetBMI' to match pascal case naming rules, consider using 'GetBmi'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4835315)